README.md

```diff
@@ -1,35 +1,46 @@
+# Morning challenge - Build a request module
+This week in your project you will be making API calls from the backend. In the past few weeks you have been making API calls from the frontend with the XMLHttpRequest object and methods. Now that you are in the backend we would like you to learn how to make similar requests using the Node core [http module](https://nodejs.org/api/http.html). This workshop is intended to familiarise you with how backend requests are made, and practice using callbacks.
 
 ### Time Required: 1 hour
   - 10 mins to read challenge instructions
   - 40 mins to complete the challenge in pairs
   - 10 mins to regroup and talk over the solution
 
 ### Learning outcome
-To become familiar with the Request module and the Node core [http module](https://nodejs.org/api/http.html).
+To become familiar with making API calls from the backend using the Node core [http module](https://nodejs.org/api/http.html).
 
-### What is the Node Request module?
-Request is a very popular module that helps to make HTTP requests as simple as possible and it is built around the Node HTTP core module. [Here](http://stackabuse.com/the-node-js-request-module/) is a brilliant article on the Request module.
+### When would I need to make an API call from the backend?
+Example 1 - I would like to make a HTTP request to the [GitHub API](https://developer.github.com/v3/) to get my profile photo URL.
 
-### When would I need to use the request module?
-Example 1 - I would like to GET my profile-photo.png from the database to display on my homepage.
+Example 2 - I would like to make a HTTP request to the [TFL API](https://api.tfl.gov.uk/), to find out which buses go between London Bridge and Finsbury Park.
 
-Example 2 - I would like to use the Request module to make a HTTP request to the [TFL API](https://api.tfl.gov.uk/), to find out which busses go between London Bridge and Finsbury Park.
+### The challenge
 
-### Your challenge
-Create a function called **myRequest** that has similar set of features to the Node Request module.
+Create a function called `myRequest` which makes a request to an API url using an http GET request.
 
-The npm Request module simply creates a wrapper around the Node core http module. Your task it to create your own request function `myRequest` that performs in the same way as the Request npm module, which is used in the app.js file in this repo.
+It should take two arguments: the url to make the request to, and a callback function.
+
+The callback function should take two arguments:
+
+  1. error (null if no error occurred; an error object if an error occurred)
+  2. response (Object: includes the body & statusCode of the request)
```

Comment:
It would be helpful to add a hint that you should be constructing your own response object here

Reply:
They don't _have_ to make a new object—the Node `response` object already has the `statusCode` so they could just add the `body` to it and use that

Reply:
Probably deserves some kind of note/hint, it's something we both got stuck on trying out the new workshop

Reply:
I guess this is why the old workshop had the callback take 3 arguments: `(error, response, body)`

Reply:
I've added a [hint](https://github.com/foundersandcoders/mc-request-module-workshop/pull/8/commits/1774c5c438a3338130fcd2efa4634071aee071e1)